### PR TITLE
sslscan: Install man page.

### PR DESCRIPTION
**Pull request by dsXLII**
Does what it says on the tin - install the man page, not just the application.

### CODE REVIEW

File: Library/Formula/sslscan.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ class Sslscan < Formula
   def install
     system "make"
     bin.install "sslscan"
+    man1.install "sslscan.1"
```
Comment (bfontaine) [inline Library/Formula/sslscan.rb]: It seems like we could use `system "make", "install", "BINPATH=#{bin}", "MANPATH="#{man}/"`:
https://github.com/rbsec/sslscan/blob/d0fdf8c38389d9d58b07e50ddd8f479e09a9b5d7/Makefile#L54

**Comment by dsXLII** [comment]
I'm pretty new to Homebrew, so I don't know if there's a preference one way or the other. I did it in the most obvious way, to me, from reading the docs; if that's wrong, cool, just let me know.

**Comment by dunn** [comment]
Yeah, we try to use Makefile when possible. Thanks!

**Comment by dsXLII** [comment]
So, when I make that change, the installer says it's installing 5 files (instead of 7), and neither the binary nor the man page are installed to /usr/local/Cellar (or anywhere, as far as I can tell). It looks like the environment params I'm specifying are being passed in, but maybe their Makefile doesn't honor them?

`
is-mac001:Formula davsmi$ brew install sslscan
==> Downloading https://github.com/rbsec/sslscan/archive/1.11.1-rbsec.tar.gz
Already downloaded: /Library/Caches/Homebrew/sslscan-1.11.1.tar.gz
==> make install BINPATH=/usr/local/Cellar/sslscan/1.11.1/bin MANPATH=/usr/local/Cellar/sslscan/1.11.1/share/man
🍺  /usr/local/Cellar/sslscan/1.11.1: 5 files, 60K, built in 2 seconds
`

Suggestions on how to troubleshoot would be welcomed, as I've never been much of a developer, and am still pretty new to Homebrew.

**Comment by bfontaine** [comment]
Just checked; they have an `INSTALL` file that causes `make install` not to do anything (``make: `install' is up to date.``). To circumvent this, just remove the file before running `make install`:

```rb
  def install
    rm "INSTALL"
    system "make"
    system "make", "install", "BINPATH=#{bin}", "MANPATH=#{man}/"
  end
```

It should work as expected:
```
==> Downloading https://github.com/rbsec/sslscan/archive/1.11.1-rbsec.tar.gz
Already downloaded: /Library/Caches/Homebrew/sslscan-1.11.1.tar.gz
==> make
==> make install BINPATH=/usr/local/Cellar/sslscan/1.11.1/bin MANPATH=/usr/local/Cellar/sslscan/1.11.1/share/man/
🍺  /usr/local/Cellar/sslscan/1.11.1: 7 files, 128K, built in 1 second
```

**Comment by dsXLII** [comment]
Pull request updated. Thanks for helping the newbie. :)

File: Library/Formula/sslscan.rb [hunk 2]
```diff
@@ -6,18 +6,12 @@ class Sslscan < Formula
   sha256 "0631713b16cea51df49b9666aa17e742e8177d79e85bdb13f66105657c98f169"
   head "https://github.com/rbsec/sslscan.git"
 
-  bottle do
-    cellar :any
-    sha256 "fc1966c79c103f55cd09b1e9e73b9fa813e115364e2d88d9c5d5a2fc5353fcb2" => :el_capitan
-    sha256 "a550ac845f1b4da274aea5637bc09c5492536bea05f446a0e5aa3dd8be45c35c" => :yosemite
-    sha256 "eda25606cd361b28afbcd64397933d6d22023b821a716aad99f512cc7ef0e838" => :mavericks
-  end
```
Comment (bfontaine) [inline Library/Formula/sslscan.rb]: Please leave this block as is; our bot will update it. To build the formula from source use `brew install --build-from-source` or `brew install -s`.

**Comment by apjanke** [comment]
Case-insensitive filesystem strikes again. The `install` target should be marked as `.PHONY` to prevent this from happening. I've submitted an upstream PR for this – rbsec/sslscan#72 – as well as one to add `PREFIX` support (so you don't have to set `BINPATH` and `MANPATH` separately, and avoid an issue with overloading `$MANPATH`'s meaning) – rbsec/sslscan#73.

**Comment by apjanke** [comment]
Tests look good, and I tested locally (to make sure the man page really made it in.) Looks good.

Merged. Thank you for your contribution to Homebrew!

**Comment by rbsec** [comment]
Both of the pull requests have been merged, so hopefully that'll make it a little simpler for you.

~rbsec

**Comment by apjanke** [comment]
Yep! Thanks rbsec.

Note to Homebrew maintainers: when next `sslscan` release comes out, we can remove the sslscan-specific logic from the formula and go back to a plain `make` / `PREFIX` install method. I added a comment in the formula to that effect.

**Comment by bfontaine** [comment]
Thanks @apjanke :smiley:

**Pull request closed without merging**